tests/bundle/SystemInfo/Collector/ServicesSystemInfoCollectorTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Tests\Bundle\SystemInfo\SystemInfo\Collector;
+
+use Ibexa\Bundle\SystemInfo\SystemInfo\Collector\ServicesSystemInfoCollector;
+use Ibexa\Bundle\SystemInfo\SystemInfo\Value\ServicesSystemInfo;
+use Ibexa\SystemInfo\Service\Service;
+use Ibexa\SystemInfo\Service\ServiceProviderInterface;
+use PHPUnit\Framework\TestCase;
+
+class ServicesSystemInfoCollectorTest extends TestCase
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject
```

Comment:
Incomplete annotation: this property represents both `MockObject` and `ServiceProviderInterface` interface.